style-editor.scss

```diff
@@ -12,6 +12,12 @@ when Gutenberg supports styling those variations more intuitively.
 @import "sass/variables-site/colors";
 @import "sass/mixins/mixins-master";
 
+@font-face {
```

Comment:
It's probably wise to add a comment to this hack. :)

Reply:
Good thinking. 😄